packages/shared-ux/error_boundary/src/services/error_boundary_services.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, useContext, useMemo } from 'react';
+
+import { KibanaErrorBoundaryServices } from '../../types';
+import { KibanaErrorService } from './error_service';
+
+const Context = React.createContext<KibanaErrorBoundaryServices | null>(null);
+
+/**
+ * A Context Provider for Jest and Storybooks
+ */
+export const KibanaErrorBoundaryDepsProvider: FC<KibanaErrorBoundaryServices> = ({
+  children,
+  reloadWindow,
+  errorService,
+}) => {
+  return <Context.Provider value={{ reloadWindow, errorService }}>{children}</Context.Provider>;
+};
+
+/**
+ * Kibana-specific Provider that maps dependencies to services.
+ */
+export const KibanaErrorBoundaryProvider: FC = ({ children }) => {
+  // control side-effects of rendering with useMemo
+  const reloadWindow = useMemo(() => () => window.location.reload(), []);
```

Comment:
Thank you for these suggestions! 

Since `KibanaErrorBoundaryServices` is just an interface, I'll go with:
```
  const value: KibanaErrorBoundaryServices = useMemo(
    () => ({
      reloadWindow: () => window.location.reload(),
      errorService: new KibanaErrorService(),
    }),
    []
  );
```